### DISCUSSION TITLE
PCL的自定义主页：local:Mycard控件支持更改卡片颜色吗

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4666] [讨论]
local:Mycard控件支持更改卡片颜色吗

**Comment by MoYuan-CN** [comment]
支持是支持（使用 `Background` 属性），但是这样做会非常奇怪
![image](https://github.com/user-attachments/assets/e8d9c0c8-d147-4e5f-8a5a-98a166ec1565)
